test/dialect/mssql/test_reflection.py

```diff
@@ -568,10 +568,166 @@ def test_indexes_with_filtered(self, metadata, connection):
 
         t2 = Table("t", MetaData(), autoload_with=connection)
         idx = list(sorted(t2.indexes, key=lambda idx: idx.name))[0]
+        self.assert_compile(
+            CreateIndex(idx),
+            "CREATE NONCLUSTERED INDEX idx_x ON t (x) WHERE ([x]='test')",
```

Comment:
this was an existing test that needed to be updated to include the NONCLUSTERED keyword in the DDL statement since the reflected index now contains that info